modular-particle-system/package.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
you need to expose something like

```
"main": "lib/index.js",
"module": "lib/index.js",
"types": "lib/index.d.ts",
```

So that NPM and IDEs would find it

Reply:
This was actually something that confused me, because in case of this library there is no "index file".

Importing already works without this, but you need to specify the file paths like so:
![image](https://user-images.githubusercontent.com/37183581/156918624-9e041e8b-8b59-4216-ab5e-6f9cb3b8cbd6.png)

To me it seems that we don't need these unless we want to add some deliberate index file, which would then allow importing stuff with this kind of syntax:
![image](https://user-images.githubusercontent.com/37183581/156918741-d6af9f45-8c4a-48e2-a1b4-6c549c7ae492.png)

What do you think @Nikolsi ?